utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -346,18 +346,6 @@ int main(int argc, char **argv)
     if(testCase == 82)
         CHECK(hipHostMalloc(&roiPtrInputCropRegion, 4 * sizeof(RpptROI)));
 
-    RpptROI *cropRoi, *patchRoi;
-    if(testCase == 33)
-    {
-        CHECK(hipHostMalloc(&cropRoi, batchSize * sizeof(RpptROI)));
-        CHECK(hipHostMalloc(&patchRoi, batchSize * sizeof(RpptROI)));
-    }
```

Comment:
Please revert this changes
There is some problem with merge

I will again review the test suite files, once the issues are fixed

Reply:
Done